grammars/level5-Additions.lark

```diff
@@ -17,6 +17,7 @@ assign: var _IS textwithspaces
 error_print_no_quotes: _PRINT (textwithoutspaces | list_access | var_access) (_SPACE (textwithoutspaces | list_access | var_access))*  -> error_print_nq
 
 // new commands for level 5
+ifpressed: _IF SMALL_LETTER_OR_NUMERAL _IS _PRESSED _SPACE* _if_less_command
```

Comment:
You can add `_SPACE*` to the definition of  `_PRESSED` that way we don't have to use that terminal inside the production rule, keeping grammars cleaner!

Reply:
Thanks @jpelay for noticing that!! commit [Update level5-Additions.lark - Remove unnecessary space in ifpressed](https://github.com/hedyorg/hedy/pull/3483/commits/11098f2b78e325bdc9df6346684226cfc64f2968) resolves this, This was not supposed to be here, SPACE? is already in _PRESSED.